modules/backend/clinic/staff/doctor/i_doctor.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "modules/backend/common/enumerations/specialization.hpp"
+#include "modules/backend/common/personal_data/personal_data.hpp"
+
+class iDoctor
+{
+  public:
+    virtual ~iDoctor() = 0;
+    virtual PersonalData& GetPersonalData() = 0;
```

Comment:
I think we should stick to our convention of naming functions starting from the lowercase letter and All types names starting from uppercase.

This creates some problems for naming interfaces. I don't know what to do with iDoctorForExample. Maybe BaseDoctor?

Reply:
This is my proposition, more information here (hope link works as intened):
https://github.com/coders-school/desktop-business-app/pull/121/files#diff-aee7177b08a803cf66c7e5c35b817aea2232da4c6c81cfda2160f7b9a02e0249

Reply:
I am used to 'i' prefix for the interfaces. We could go with something different. 
I think it will be missinterpreted. If we will have a collection of BaseDoctors someone might want to create another collection for specialized type of doctors.